sharktank/sharktank/layers/configs/llm_configs.py

```diff
@@ -47,41 +48,19 @@ class LlamaHParams:
     attention_layer_norm_rms_epsilon: float
     attention_head_count_kv: int
 
-    # Deepseek Multi-Latent Attention config
-    q_lora_rank: Optional[int] = None
```

Comment:
So I would keep these configs around, just use the `dict` trick to avoid specifying all of them when some are optional